src/node_config.rs

```diff
@@ -191,6 +235,67 @@ pub fn cli_parse() -> CliConfig {
             .help("Disable LAN discovery"))
         .get_matches();
 
+    match matches.subcommand() {
+        ("config", Some(m)) => run_config(m),
+        _ => run_args(&matches),
+    }
+}
+
+/// Parse settings from a saved file.
+fn parse_config(config_path: String) ->Result<NodeConfig, ()> {
+    let mut settings = Config::default();
+
+    let default_str = r#"
```

Comment:
How about using [set_defaoult](https://docs.rs/config/0.9.1/config/struct.Config.html#method.set_default)? It seems more typesafe.

Reply:
Or we can even use https://serde.rs/attr-default.html